src/sdk/pynni/nni/_graph_utils.py

```diff
@@ -0,0 +1,446 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
It seems you rewrite the logic of `graph_utils.py` in pytorch. just wondering is it possible to directly use the output of `graph_utils.py` in pytorch?

Reply:
each node in graph_utils.py is directly mapped a single traced C node, the main logic of the graph for shape inference is focusing on grouping multiple nodes together.  Directly use the output of graph_utils.py introduces additional dependencies and save no efforts.